### PR TITLE
docs: clarify wording in architecture overview

### LINKED ISSUE #22158: Documentation error which says NgModule instead of BrowserModule
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

In Architecture Overview --> Angular Libraries 
as in the below screen print it says "You also import NgModules from Angular libraries using JavaScript import statements:" which is in contradiction with the example shown in the code
image

## Expected behavior

The description should have been "You also import BroserModul from Angular libraries using JavaScript import statements:" to inline with the code example given

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

### LINKED ISSUE #23463: Semantics of Arguments Passed to Constructor of Component
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[X] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior
In the URL: https://angular.io/guide/architecture-components
at the place where the "class HeroListComponent"  is defined, the explanation for the semantics of passing the argument "private service: HeroService" to the constructor method is missing.
Someone coming from the OOP background, they think that it is just a formal parameter and expects it to be initialized with the instance of the object created with new operator. 

## Expected behavior
Looks like, in Angular, by virtue of specifying it as a formal parameter to constructor method it automatically does the following:
private service: HeroService;
constructor() {
   service = new HeroService();
}
If this is the case, in case some parameters to be passed while constructing as HeroService(p1,p2,p3), how it is done when it can be done in the constructor() syntax.

## Minimal reproduction of the problem with instructions
None

## What is the motivation / use case for changing the behavior?
Not Applicable

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:  Not Applicable
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:  Not Applicable
- Node version: XX  
- Platform:  

Others:  Not Applicable

### PR DESCRIPTION
**Pull request by jbogarthyde**
Closes #23463
Closes #22158

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A
Closes #23463
Closes #22158

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by jbogarthyde** [comment]
See PR #24475

**Comment by jbogarthyde** [comment]
@jenniferfell @mhevery What's holding this up? Minor changes.

**Comment by mhevery** [comment]
You need to put `PR action: merge` if you want the care taker to look at this. Otherwise it just sits there. 

Working on it ...

**Review by mhevery — APPROVED** [review]

**Comment by jbogarthyde** [comment]
@jenniferfell waiting for your approval

File: aio/content/guide/architecture-components.md [hunk 1]
```diff
@@ -8,9 +8,10 @@ A _component_ controls a patch of screen called a *view*. For example, individua
 * The list of heroes.
 * The hero editor.
 
-You define a component's application logic&mdash;what it does to support the view&mdash;inside a class. The class interacts with the view through an API of properties and methods.
+You define a component's application logic&mdash;what it does to support the view&mdash;inside a class.
+The class interacts with the view through an API of properties and methods.
 
-For example, the `HeroListComponent` has a `heroes` property that returns an array of heroes that it acquires from a service. `HeroListComponent` also has a `selectHero()` method that sets a `selectedHero` property when the user clicks to choose a hero from that list.
+For example, the `HeroListComponent` has a `heroes` property that holds an array of heroes. It also has a `selectHero()` method that sets a `selectedHero` property when the user clicks to choose a hero from that list. The component acquires the heroes from a service, which is a TypeScript [parameter property[(http://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties) on the constructor. The service is provided to the component through the dependency injection system.
```
Comment (trotyl) [inline aio/content/guide/architecture-components.md]: Incorrect brackets at `[parameter property[`

Comment (jbogarthyde) [reply aio/content/guide/architecture-components.md]: Thanks -- Misko had already closed this, but I noticed the error and fixed it in a new PR
https://github.com/angular/angular/pull/24825

Comment (jbogarthyde) [reply aio/content/guide/architecture-components.md]: This is corrected by new PR https://github.com/angular/angular/pull/24825, currently awaiting merge.

**Pull request closed without merging**